src/Plugin/migrate/source/OntologyWordDetails.php

```diff
@@ -69,6 +62,27 @@ public function fields() : array {
     return [];
   }
 
+  /**
+   * Gets the ontology ids that are included to the migration.
+   *
+   * Set ontology ID limit using 'helfi_tpr.limit_ontology_words:ids' config.
+   * If the config is not set, $defaultOntologyIdsLimit is used.
+   *
+   * If given an empty array, migrate does not limit by ontology ids.
+   *
+   * @return array
+   *   List of ontology ids.
+   */
+  public function getOntologyIdsLimit(): array {
+    if ($config = \Drupal::configFactory()->get('helfi_tpr.limit_ontology_words')) {
+      if ($ids = $config->get('ids')) {
+        return $ids;
```

Comment:
This should probably be changed to `$this->defaultOntologyIdsLimit = $ids;` or just remove the $defaultOntologyIdsLimit variable.

Reply:
Removed the variable altogether.